shared/src/main/scala/implicits.scala

```diff
@@ -17,16 +17,15 @@
 package fetch
 
 import cats.{Eval, MonadError}
-import cats.std.FutureInstances
+import cats.instances.FutureInstances
 import scala.concurrent.{Promise, Future, ExecutionContext}
 
 object implicits extends FutureInstances {
   implicit def fetchFutureFetchMonadError(
-      implicit ec: ExecutionContext,
-      ME: MonadError[Future, Throwable]
+      implicit ec: ExecutionContext
   ): FetchMonadError[Future] = new FetchMonadError[Future] {
     override def runQuery[A](j: Query[A]): Future[A] = j match {
-      case Sync(e) => ME.pureEval(e)
+      case Sync(e) => pure(e.value)
```

Comment:
@raulraja thanks for the hint!
Is is possible that this change is causing tests to fail?
```
[info] - Every level of joined concurrent fetches is combined and batched *** FAILED ***
[info]   (1,1,6) did not equal (3,3,6) (FetchTests.scala:481)
[info] - The product of concurrent fetches of the same type implies everything fetched in a single batch *** FAILED ***
[info]   (1,1,4) did not equal (2,1,4) (FetchTests.scala:457)
[info] - Every level of sequenced concurrent of concurrent fetches is batched *** FAILED ***
[info]   (5,5,19) did not equal (3,3,19) (FetchTests.scala:508)
```